plugins/org.eclipse.xtext.builder/src/org/eclipse/xtext/builder/nature/messages.properties

```diff
@@ -1,3 +1,3 @@
-NatureAddingEditorCallback_MessageDialog_Title=Add Xtext Nature
-NatureAddingEditorCallback_MessageDialog_Msg0=Do you want to add the Xtext nature to the project '
+NatureAddingEditorCallback_MessageDialog_Title=Configure Xtext
+NatureAddingEditorCallback_MessageDialog_Msg0=Do you enable Xtext for the project '
```

Comment:
Incomplete sentence: should be
```
Do you want to enable Xtext for the project ...
```
or, in order to be more consistent with the terms used in the context menu,
```
Do you want to convert ... to an Xtext project?
```